### DISCUSSION TITLE
Slow Connection Speeds

### DISCUSSION THREAD

**Discussion by kyleinprogress** [haugene/docker-transmission-openvpn#2011] [Q&A] [answered]
Hey Team,

Does anyone have any thoughts/recommendations on how to increase connection speeds? I have 1 Gig Fiber and when using other services (NZB, Rclone -> G Drive) I routinely see 800-900 Mbps. So I know the network is working fine.

In Transmission though, I seldom get above 100 Mbps and often only max out around 30-40 Mbps. I'm on a few private trackers with dedicated seedboxes, so I'm wondering if it's just a result of running things over a VPN. 

I'm on PIA as my VPN provider.

**Comment by pkishino** [comment, marked as answer]
Vpn will definitely put a damper on it.. are you using udp or tcp? Have you enabled/disabled utp in transmission? You could try and run a Speedtest from the cli of the container using this:
https://www.speedtest.net/apps/cli

**Reply by kyleinprogress** [reply]
Good call. Looks like it's just the VPN slowing things down. 

From the docker host:
```
Latency:     2.22 ms   (0.46 ms jitter)
Download:   939.92 Mbps (data used: 900.7 MB)                               
Upload:   939.27 Mbps (data used: 991.1 MB)                               
Packet Loss:     0.0%
```
From the container:
```
Latency:    53.01 ms   (8.31 ms jitter)
Download:    39.41 Mbps (data used: 61.4 MB)                               
Upload:    20.54 Mbps (data used: 32.2 MB)                               
Packet Loss:     0.7%
```
As far as UDP vs TCP, I believe it's UDP since I'm using a standard config from PIA. I think they default to UDP unless you specify TCP specifically.

**Reply by pkishino** [reply]
yeah, I guess you could try a few different servers etc to find best one
